bench/BenchAll.hs

```diff
@@ -425,7 +432,15 @@ main = do
       , bench "groupBy (>)"    $ nf (L.groupBy (>)) zeroes
       ]
     , bgroup "findIndex"
-      [ bench "findIndices"    $ nf (sum . S.findIndices even) byteStringData
-      , bench "find"           $ nf (S.find (>= 9998)) byteStringData
+      [ bench "findIndices"    $ nf (sum . S.findIndices (==  129)) byteStringData
+      , bench "find"           $ nf (S.find (>= 198)) byteStringData
```

Comment:
Err, what's happened here? What was the reason to change existing benchmarks?

Reply:
9998 was out of bounds and so gave a warning (it was a mindless choice first time). Using `findIndices even` was not really testing the inner loop of findIndex very well so I decided to alter slightly to test the right thing.

Reply:
`S.findIndices (== 129)` could hit a rewrite rule and become `elemIndices 129`. 
https://github.com/haskell/bytestring/blob/eea70ff9515705a03d89b40da7d37d10152488c1/Data/ByteString.hs#L1387

Please use something like `\x -> x == 129 || x == 130` or similar.

Reply:
@Boarders this is the last outstanding bit, as far as I'm aware.

Reply:
Now changed, I also changed the name of the benchmark group to `findIndex_` so it can more easily be run on its own.